pkg/cmd/server/start/start_master.go

```diff
@@ -474,16 +474,15 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	}
 
 	// verify we can connect to etcd with the provided config
-	// TODO(post-1.6.1-rebase): uncomment etcd3 support
```

Comment:
@sttts @ncdc up to you guys if you want to leave this disabled, would have to disable the etcd3 test since without the V3 client this check fails.